src/mainviews/create-job.tsx

```diff
@@ -191,6 +191,26 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
     }
   };
 
+  const handleNumericInputChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const target = event.target;
+
+    const parameterNameIdx = parameterNameMatch(target.name);
```

Comment:
This code is copied from the code for setting parameters, which have user-provided names and values. Your change is intended to add input fields with only user-provided values.